WordPress/Classes/Utility/Migrations/87-88/BlogToBlogMigration87to88.swift

```diff
@@ -0,0 +1,24 @@
+import CoreData
+
+class BlogToBlogMigration87to88: NSEntityMigrationPolicy {
+    override func createRelationships(forDestination dInstance: NSManagedObject, in mapping: NSEntityMapping, manager: NSMigrationManager) throws {
+        try super.createRelationships(forDestination: dInstance, in: mapping, manager: manager)
+
+        guard let sourceBlog = manager.sourceInstances(forEntityMappingName: "BlogToBlog", destinationInstances: [dInstance]).first else {
+            return
+        }
+
+        let editor: MobileEditor
```

Comment:
I think migrations generally should not have external dependencies and use plain strings instead of `MobileEditor` or `GutenbergSettings`. Those objects might change in the future and we want migrations to be always the same.

Reply:
Right! That makes sense. Changed on https://github.com/wordpress-mobile/WordPress-iOS/pull/12189/commits/5c4e23be4782864d2c6927b8c4f3868c9e0f4bfc